css/style.css

```diff
@@ -189,11 +188,16 @@ header {
 	border: 2px solid #539dab;
 	background-color: #539dab;
 	color: #F1F1F2;
-    border-radius: 25px;
-    width: 140px;
+	border-radius: 25px;
+	width: 140px;
 }
 
 .startOver:hover {
 	background-color: #1995AD;
 	border: 2px solid #1995AD;
-}
\ No newline at end of file
+}
+
+.hidden {
+	display: none;
+}
+
```

Comment:
I got rid of the .hidden2, and .hidden3 selectors/classes. Since classes are reusable and they were doing the same thing, we can just use a single .hidden class. We could use jQuery to remove/add that class to elements, and it would function the same way as .show()/.hide(), but those functions have a few some extra logic in them that we definitely want to use. 

This class is a good way to have everything hidden when the page loads; however, at some point you might want to consider having a function that calls .hide() on these elements, rather than using a .hidden class. That way you can call it in two spots: (1) document.ready (for the initial page load), and (2) as part of the New Game button functionality. But you can save that for when you get there.